include/grpc++/impl/codegen/async_stream.h

```diff
@@ -225,7 +275,8 @@ class ClientAsyncWriter final : public ClientAsyncWriterInterface<W> {
   Call call_;
   CallOpSet<CallOpSendInitialMetadata> init_ops_;
   CallOpSet<CallOpRecvInitialMetadata> meta_ops_;
-  CallOpSet<CallOpSendMessage> write_ops_;
+  CallOpSet<CallOpSendInitialMetadata, CallOpSendMessage, CallOpClientSendClose>
```

Comment:
how about finish_ops_ for AsyncServerWriter and AsyncServerReaderWriter? Its ops are also included in write_ops.